plugins/topology/src/utils/resource-utils.ts

```diff
@@ -0,0 +1,40 @@
+import * as _ from 'lodash';
+import { OverviewItem } from '../types/topology-types';
+import { K8sResourceKind } from '../types/types';
+import { WORKLOAD_TYPES } from './topology-utils';
+
+export const validPod = (pod: K8sResourceKind) => {
+  const owners = pod?.metadata?.ownerReferences;
+  const phase = pod?.status?.phase;
+  return _.isEmpty(owners) && phase !== 'Succeeded' && phase !== 'Failed';
+};
+
+const isStandaloneJob = (job: K8sResourceKind) =>
+  !_.find(job.metadata?.ownerReferences, owner => owner.kind === 'CronJob');
```

Comment:
Can we utilize the find method in the javascript instead of the library

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/find